app/controllers/check_records/omniauth_callbacks_controller.rb

```diff
@@ -7,6 +7,7 @@ def dfe_bypass
     redirect_to check_records_root_path unless CheckRecords::DfESignIn.bypass?
     @dsi_user = DsiUser.create_or_update_from_dsi(request.env["omniauth.auth"])
     @dsi_user.begin_session!(session)
+    session[:dsi_user_token_expiry] = 2.hours.from_now.to_i
```

Comment:
We could make this part of `#being_session!`

Reply:
That's a hangover from another service that I based this implementation on. Not particularly attached to the approach, but I think any session handling related to this callback should be co-located, whether it's here or in that method. Happy for the method to be removed and its logic added here if you prefer 👍🏼